rest-service/manager_rest/utils.py

```diff
@@ -184,6 +187,14 @@ def all_tenants_authorization():
     )
 
 
+def tenant_specific_authorization(tenant, resource_name, action='list'):
```

Comment:
comment.. verify user have permission to exec this action in current tenant on that resource..